src/java_jstat

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
+#
+# igcollect - jvm_jstat
+#
+# Copyright (c) 2017, InnoGames GmbH
+
+from time import time
+from argparse import ArgumentParser
+from subprocess import check_output
+
+
+def main():
+    args = vars(parse_args())
+
+    if args['option'] is None:
+        args['option'] = get_options(args['jvm_install_path'])
+
+    metrics = []
+    for vm in args['vmid']:
+        for option in args['option']:
+            metrics.append(get_metrics(
+                args['jvm_install_path'],
+                args['prefix'],
+                option,
+                vm,
+                args['interval'],
+                args['count']
+            ))
+
+
+def parse_args():
+    parser = ArgumentParser(
+        description='java_jstat graphite metrics collector',
+        epilog=(
+            'Script to collect metrics using jstat about running JVMs and'
+            'transform them into graphite format.'
+        )
+    )
+    parser.add_argument(
+        '--option',
+        nargs='*',
+        help='jstat option to collect e.g. gcutil or all if specified'
+    )
+    parser.add_argument(
+        '--vmid',
+        nargs='+',
+        help='Java VM id to gather metrics from, check jps tool'
+    )
+    parser.add_argument(
+        '--interval',
+        default='1',
+        help='Sampling interval in seconds'
+    )
+    parser.add_argument(
+        '--count',
+        default=4,
+        help='Number of samples before terminating'
+    )
+    parser.add_argument(
+        '--jvm-install-path',
+        help='JVM installation path e.g. /usr/lib/jvm/jdk-8-oracle-x64',
+        required=True
+    )
+    parser.add_argument(
+        '--prefix',
+        default='jvm.jstat',
```

Comment:
The commit message says "java_jvm".  The script is called "java_jstat".  And the default prefix is "jvm.jstat".  Would you stick with one of them?